### PR TITLE
Fix bug 1476617 - [Translate.Next] Add an Editor.

### PR DESCRIPTION
**Pull request by adngdb**
This is a big milestone, as this PR solves a few problems: 

- handling CSRF token in the frontend;
- a basic updating for data in the server;
- annoying logging during tests. 

And it of course adds a very basic Editor with the current 3 buttons.

### CODE REVIEW

File: frontend/src/modules/editor/components/Editor.test.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    locale: 'kg',
+};
+
+describe('<EditorBase>', () => {
+    it('renders correctly', () => {
+        const wrapper = shallow(<EditorBase
+            activeTranslation={ TRANSLATION }
+            navigation={ NAVIGATION }
+            selectedEntity={ SELECTED_ENTITY }
+        />);
```
Comment (mathjazz) [inline frontend/src/modules/editor/components/Editor.test.js]: This block is repeated 4 times in this file. Can we factor that out?

File: frontend/src/modules/editor/actions.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        payload.append('plural_form', '0');
+        payload.append('original', original);
+
+        // Get the CSRF value.
+        let csrfToken = '';
+        const rootElt = document.getElementById('root');
+        if (rootElt) {
+            csrfToken = rootElt.dataset.csrfToken;
+        }
+        payload.append('csrfmiddlewaretoken', csrfToken);
```
[the remark this answers is not in the document]
Comment (adngdb) [reply frontend/src/modules/editor/actions.js]: Yes, of course. However, I wanted to not do that in this PR and instead wait until there are three places where we call the API, then refactor those calls into something else. At the moment, I fear that if I refactor this I'll do it wrong.

File: frontend/src/App.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#app {
+    display: flex;
+}
```
[the remark this answers is not in the document]
Comment (adngdb) [reply frontend/src/App.css]: Ah, CSS Grid does scare me I have to admit. I've been trying to use it a bunch and have been struggling. I find it a very complex feature, all great that it is. Also, we'll probably have to add some fallbacks anyway, I don't think all browsers we support do handle grid.

[the remark this answers is not in the document]
Comment (mathjazz) [reply frontend/src/App.css]: There are a lot of technologies in this rewrite that scare me, so IMHO this is not a very strong argument. :)

**Pull request merged**